unit-tests/test-coldlaunch.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+
+let assert = require('chai').assert;
+let coldlaunch = require('../lib/phases/cold-launch.js');
+
+let OPTIONS = {
+  'app': 'communications',
+  'entryPoint': 'contacts',
+  'homescreen': 'homescreen.gaiamobile.org',
+  'system': 'system.gaiamobile.org',
+  'memoryDelay': 10000,
+  'launchDelay': 20000,
+  'metrics': 'metrics.ldjson',
+  'runs': 30,
+  'timeout': 30000,
+  'retries': 3
+};
+
+let COLDLAUNCH_TITLE = 'Cold Launch: ' + OPTIONS.app;
+let COLDLAUNCH_NAME = 'coldlaunch';
+let COLDLAUNCH_START_MARK = 'appLaunch';
+
+// cold-launch unit tests
+describe('cold-launch', function() {
+
+  describe('constructor', function() {
+
+    var myColdLaunch = new coldlaunch(OPTIONS);
+
+    it('coldlaunch', function() {
+      assert.isObject(myColdLaunch);
+      assert.instanceOf(myColdLaunch, coldlaunch);
+    });
+
+    // options
+    it('options', function() {
+      assert.propertyVal(myColdLaunch, 'options', OPTIONS);
+    });
+
+    // title
+    it('title', function() {
+      assert.property(myColdLaunch, 'title');
+      assert.strictEqual(myColdLaunch.title, COLDLAUNCH_TITLE);
+    });
+
+    // name
+    it('name', function() {
+      assert.property(myColdLaunch, 'name');
+      assert.strictEqual(myColdLaunch.name, COLDLAUNCH_NAME);
+    });
+
+    // start mark
+    it('start mark', function() {
+      assert.property(myColdLaunch, 'START_MARK');
+      assert.strictEqual(myColdLaunch.START_MARK, COLDLAUNCH_START_MARK);
+    });
+
+    // launches buffered
+    it('launches buffered', function() {
+      assert.property(myColdLaunch, 'launchesBuffered');
+      assert.strictEqual(myColdLaunch.launchesBuffered, 0);
+    });
+
+    // capture
+    it('capture', function() {
+      assert.isFunction(myColdLaunch.capture);
+    });
+
+    // captureEntryMetadata
+    it('captureEntryMetadata', function() {
+      assert.isFunction(myColdLaunch.captureEntryMetadata);
+    });
+
+    // launch
+    it('launch', function() {
+      assert.isFunction(myColdLaunch.launch);
+    });
+
+    // prime
+    it('prime', function() {
+      assert.isFunction(myColdLaunch.prime);
+    });
+
+    // preventBufferOverflow
+    it('preventBufferOverflow', function() {
+      assert.isFunction(myColdLaunch.preventBufferOverflow);
+    });
+
+    // testRun
+    it('testRun', function() {
+      assert.isFunction(myColdLaunch.testRun);
+    });
+
+    // prime
+    it('prime', function() {
+      assert.isFunction(myColdLaunch.prime);
+    });
+
+    // closeApp
+    it('closeApp', function() {
+      assert.isFunction(myColdLaunch.closeApp);
+    });
+
+    // retry
+    it('retry', function() {
+      assert.isFunction(myColdLaunch.retry);
+    });
+
+    // handleRun
+    it('handleRun', function() {
+      assert.isFunction(myColdLaunch.handleRun);
+    });
+
+    // now the properties and methods inherited from phase
```

Comment:
I see now that Phase also has these same tests. Let's remove them from ColdLaunch.